articles/machine-learning/how-to-configure-auto-train.md

```diff
@@ -339,9 +339,12 @@ No&nbsp;criteria | If you do not define any exit parameters the experiment conti
 `max_concurrent_trials`| Represents the maximum number of trials (children jobs) that would be executed in parallel. It's a good practice to match this number with the number of nodes your cluster
 
 ## Run experiment
-> [!WARNING]
+> [!NOTE]
```

Comment:
Why not keep the existing warning?

Reply:
- I have added vnet related details under warning. 
- And changed information saying "Accuracy/metric would be different in every run due to randomness" under Note (from initially warning). 
- If I had put both under Warning, (under Run exp section), it might have caused little difficulty in comprehension for user, as both are different topics.
- Considering severity of 2 information, I felt vnet related info is relatively more critical. 
- Hence added vnet related info under warning and metrics related info under NOTE

Reply:
I think someone from AutoML team should take a look and approve this change.